### PR TITLE
Fixes for CAM and ARP parsing issues in nxos_ssh.py

### PR DESCRIPTION
**Pull request by rbcollins123**
I happened to hit both of these today and saw an old issue open on one (#441) and created a new issue for discussion on the other (#633).  Attached are simple fixes for each that I patched into my local instance...

### CODE REVIEW

File: napalm/nxos_ssh/nxos_ssh.py [hunk 1]
```diff
@@ -1056,6 +1056,8 @@ def get_arp_table(self):
         for line in arp_entries.splitlines():
             if len(line.split()) == 4:
                 address, age, mac, interface = line.split()
+            elif len(line.split()) == 5:
+                address, age, mac, interface, flag = line.split()
```
Comment (itdependsnetworks) [inline napalm/nxos_ssh/nxos_ssh.py]: Does it make sense to test if the flag is one of the four options?

Comment (rbcollins123) [reply napalm/nxos_ssh/nxos_ssh.py]: I do not see the point in doing that personally unless we plan to use that value for something. Currently it is just assigned and later garbage collected.  But, if there is a use case for collecting it and using it in the code later I could do that and add appropriate testing. What did you have in mind?

Comment (rbcollins123) [reply napalm/nxos_ssh/nxos_ssh.py]: Or is your concern more that down the road some other value could land in there as Cisco changes the output some day?

Comment (dbarrosop) [reply napalm/nxos_ssh/nxos_ssh.py]: Yeah, something like:

```
address, age, mac, interface = line.split()[:4]
```

Might be more resilient.

Comment (rbcollins123) [reply napalm/nxos_ssh/nxos_ssh.py]: Fair enough!  I’ll change it here in a bit

**Pull request merged**